From 7dd4db4ed78348935f7b44aa8962dc6a2e42d25d Mon Sep 17 00:00:00 2001
From: Russ Weight <russell.h.weight@intel.com>
Date: Tue, 15 Aug 2023 09:54:18 -0700
Subject: [PATCH] security: Add intel security policy file

Add Intel standard SECURITY.md file to communicate Intel's Security
Policy.

Signed-off-by: Russ Weight <russell.h.weight@intel.com>
---
 SECURITY.md | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 SECURITY.md

diff --git a/SECURITY.md b/SECURITY.md
new file mode 100644
index 000000000000..4988e77be87d
--- /dev/null
+++ b/SECURITY.md
@@ -0,0 +1,5 @@
+# Security Policy #
+Intel is committed to rapidly addressing security vulnerabilities affecting our customers and providing clear guidance on the solution, impact, severity and mitigation.
+
+## Reporting a Vulnerability ##
+Please report any security vulnerabilities in this project **[utilizing the guidelines here](https://www.intel.com/content/www/us/en/security-center/vulnerability-handling-guidelines.html)**.
